### PR TITLE
https://issues.redhat.com/browse/ACM-14568: doo not support API token to import cluster

### PR DESCRIPTION
**Pull request by ldpliu**
https://issues.redhat.com/browse/ACM-14568

### CODE REVIEW

**Review by jc-berger — APPROVED** [review]
/lgtm

File: global_hub/global_hub_import_hub_hosted.adoc [hunk 1]
```diff
@@ -11,11 +11,11 @@ Enable `local-cluster` in the {global-hub} cluster.
 [#global-hub-importing-hosted-mode]
 == Importing a hub cluster in the hosted mode 
 
-To import an existing hub cluster that has `local-cluster` enabled, you must use the hosted mode. In the hosted mode, you only get support for import managed hub clusters that use `kubeconfig` or API tokens. {acm} uses this `kubeconfig` or API token to generate the `auto-import-secret`, which connects to the managed hub cluster. 
+To import an existing hub cluster that has `local-cluster` enabled, you must use the hosted mode. In the hosted mode, you only get support for import managed hub clusters that use `kubeconfig`. {acm} uses this `kubeconfig` to generate the `auto-import-secret`, which connects to the managed hub cluster.
```
Comment (swopebe) [inline global_hub/global_hub_import_hub_hosted.adoc]: should this be import_ed_ managed hub clusters?

File: global_hub/global_hub_import_hub_hosted.adoc [hunk 2]
```diff
@@ -11,11 +11,11 @@ Enable `local-cluster` in the {global-hub} cluster.
...
+To import an existing hub cluster that has `local-cluster` enabled, you must use the hosted mode. In the hosted mode, you only get support for import managed hub clusters that use `kubeconfig`. {acm} uses this `kubeconfig` to generate the `auto-import-secret`, which connects to the managed hub cluster. 
 
 *Note:*
 
-- Make sure this `kubeconfig` or API token is always usable. If it expires, regenerate the `auto-import-secret` in the managed hub cluster namespace with the following command: `oc create secret generic auto-import-secret --from-file=kubeconfig=./managedClusterKubeconfig -n <Managedhub Namespace>`. 
+- Make sure this `kubeconfig` is always usable. If it expires, regenerate the `auto-import-secret` in the managed hub cluster namespace with the following command: `oc create secret generic auto-import-secret --from-file=kubeconfig=./managedClusterKubeconfig -n <Managedhub Namespace>`.
```
Comment (swopebe) [inline global_hub/global_hub_import_hub_hosted.adoc]: take a look at this, the command is long and embedded in the paragraph, which is reduced UX. This prob should be moved below and coded, as we do with most commands.

**Review by swopebe — CHANGES_REQUESTED** [review]
Couple comments.

File: global_hub/global_hub_import_hub_hosted.adoc (hunk 2 above)
Comment (jc-berger) [reply global_hub/global_hub_import_hub_hosted.adoc]:
```suggestion
- Make sure this `kubeconfig` is always usable. If it expires, regenerate the `auto-import-secret` in the managed hub cluster namespace with the following command:
+
----
oc create secret generic auto-import-secret --from-file=kubeconfig=./managedClusterKubeconfig -n <Managedhub Namespace>
----
```

Comment (jc-berger) [reply global_hub/global_hub_import_hub_hosted.adoc]: @ldpliu if you like this suggestion, I'll go ahead and commit it!

File: global_hub/global_hub_import_hub_hosted.adoc (hunk 1 above)
Comment (jc-berger) [reply global_hub/global_hub_import_hub_hosted.adoc]: @ldpliu would it be as Brandi suggests? Or how it's currently written, " In the hosted mode, you only get support for import managed hub clusters that use `kubeconfig`. "

Comment (swopebe) [reply global_hub/global_hub_import_hub_hosted.adoc]: that doesn't make sense as-is if you read the line before it; we as writers can change this--you can merge to make it grammatically correct.

File: global_hub/global_hub_import_hub_hosted.adoc (hunk 2 above)
Comment (swopebe) [reply global_hub/global_hub_import_hub_hosted.adoc]: This is also a change we make as writers responsible for the doc; you can merge this. It's the same command just on another line.

**Review by swopebe — APPROVED** [review]
approving pending merge of changes

File: global_hub/global_hub_import_hub_hosted.adoc (hunk 1 above)
Comment (ldpliu) [reply global_hub/global_hub_import_hub_hosted.adoc]: How about change this line to:

`To import an existing hub cluster that has `local-cluster` enabled, you must use 'kubeconfig' to import managed hub clusters in hosted mode.`

File: global_hub/global_hub_import_hub_hosted.adoc (hunk 2 above)
Comment (ldpliu) [reply global_hub/global_hub_import_hub_hosted.adoc]: It looks good to me.

File: global_hub/global_hub_import_hub_hosted.adoc (hunk 1 above)
Comment (swopebe) [reply global_hub/global_hub_import_hub_hosted.adoc]: This is confusing.

First line: _To import an existing hub cluster_ , then second line of the same sentance says _you must use 'kubeconfig' to import managed hub clusters in hosted mode_.` So now I don't know what is imported or why. 

I think the writer needs to understand function more, then we can work on this line to help it makes sense. @jc-berger FYI.

**Review by swopebe — CHANGES_REQUESTED** [review]
Request changes for newest lines because it's not clear what we are trying to say we are importing here.

File: global_hub/global_hub_import_hub_hosted.adoc (hunk 1 above)
Comment (swopebe) [reply global_hub/global_hub_import_hub_hosted.adoc]: Use **the** `kubeconfig` **file** to import managed hub clusters in hosted mode, then you can import an existing hub cluster that has` local-cluster `enabled. (????) 

(Again, I am not sure what we are trying to say here. The short description is not clear on this either.)

For writer: **kubeconfig** is a file, so we need to refer to it as we do files; standard is in the manual:

https://www.ibm.com/docs/en/ibm-style?topic=elements-files-directories

Comment (jc-berger) [reply global_hub/global_hub_import_hub_hosted.adoc]:
```suggestion
To import an existing managed hub cluster that has `local-cluster` enabled, you must use the hosted mode. In the hosted mode, you only get support for imported managed hub clusters that use a `kubeconfig` file. {acm} uses this `kubeconfig` file to generate the `auto-import-secret`, which connects to your managed hub cluster. 
```

Comment (jc-berger) [reply global_hub/global_hub_import_hub_hosted.adoc]: @ldpliu please let me know what you think of this revised description. If you approve, I'll commit the suggestion!

Comment (ldpliu) [reply global_hub/global_hub_import_hub_hosted.adoc]: It looks good to me. Thanks

File: global_hub/global_hub_import_hub_hosted.adoc [hunk 3]
```diff
@@ -11,15 +11,19 @@ Enable `local-cluster` in the {global-hub} cluster.
...
 == Importing a hub cluster in the hosted mode 
 
-To import an existing hub cluster that has `local-cluster` enabled, you must use the hosted mode. In the hosted mode, you only get support for import managed hub clusters that use `kubeconfig` or API tokens. {acm} uses this `kubeconfig` or API token to generate the `auto-import-secret`, which connects to the managed hub cluster. 
+To import an existing managed hub cluster that has `local-cluster` enabled, you must use the hosted mode. In the hosted mode, you only get support for imported managed hub clusters that use a `kubeconfig` file. {acm} uses this `kubeconfig` file to generate the `auto-import-secret`, which connects to your managed hub cluster. 
 
 *Note:*
 
-- Make sure this `kubeconfig` or API token is always usable. If it expires, regenerate the `auto-import-secret` in the managed hub cluster namespace with the following command: `oc create secret generic auto-import-secret --from-file=kubeconfig=./managedClusterKubeconfig -n <Managedhub Namespace>`. 
+- Make sure this `kubeconfig` is always usable. If it expires, regenerate the `auto-import-secret` in the managed hub cluster namespace with the following command:
```
Comment (swopebe) [inline global_hub/global_hub_import_hub_hosted.adoc]: Make sure this `kubeconfig` is always usable.

This will need to also change...line 18

**Review by swopebe — APPROVED** [review]

File: global_hub/global_hub_import_hub_hosted.adoc (hunk 3 above)
Comment (jc-berger) [reply global_hub/global_hub_import_hub_hosted.adoc]:
```suggestion
- Make sure this `kubeconfig` file is always usable. If it expires, regenerate the `auto-import-secret` in the managed hub cluster namespace with the following command:
```

**Pull request merged**